docs/adr/adr-0022-new-icon-system.md

```diff
@@ -0,0 +1,232 @@
+---
+kind: '📌 Docs/Architecture Decision Records'
+---
+# ADR 0022: Implementing a new icon system
+
+🗓️ 2023-XX-XX · ✍️ Robert Tran, Florian Sanders
+
+## The context
+
+Historically, icons were displayed from static assets in our Clever Components.
+
+Basically, it is SVG files as source of `<img>` tags, whether it is directly coded in the component or via a property, like in the `<cc-button>` component.
```

Comment:
switching to present feels weird to me here, maybe we could use present tense for the previous sentence as well? (present tense is simpler to whenever we can, let's use it!)